spec/lib/submission_builder/ty2022/states/return_header_spec.rb

```diff
@@ -42,6 +42,25 @@
         end
       end
 
+      context "paid preparer information group" do
+        if state_code == "nj"
```

Comment:
[pebble] Could use the existing schema required mechanism instead

Reply:
I'm so sorry that I missed this comment! Added it in the follow up ticket that I will take a look at next week.

https://github.com/newjersey/affordability-pm/issues/230